site/content/docs/monitoring/monitoring.md

```diff
@@ -28,13 +28,12 @@ A picture is a thousand words:
 
 ![monitoring and alerting pipeline](/docs/images/monitoring/pipeline.png)
 
-1. A cron process picks up a check based on its schedule, say every 5 minutes. It validates that the check is not in progress at the moment to avoid race conditions.
-2. The script is put into a queue to be run from the next configured data center location. If the check is an API check and has a [setup script](/docs/api-checks/setup-teardown-scripts/), the setup script is executed. 
-3. If the check fails and "double check" is enabled, the check is retried once in the same location *and* in a different location.
```

Comment:
It would be nice if another dev can confirm, but I'm pretty sure it's no longer accurate that a check is retried in the same location and in a different location.

Reply:
It is an "or" I think